webapp/channels/src/packages/mattermost-redux/src/client/rudder.ts

```diff
@@ -11,6 +11,22 @@ import {isSystemAdmin} from 'mattermost-redux/utils/user_utils';
 
 export {rudderAnalytics};
 
+export const TrackGroupsFeature: string = 'custom_groups';
```

Comment:
nit : lets separate the feature names to constants folder, since we will be adding more of these going forward